build/build.sh

```diff
@@ -43,3 +44,20 @@ wkhtmltopdf \
   --quiet \
   output/index.html \
   output/manuscript.pdf
+
+echo "Exporting Word Docx manuscript"
```

Comment:
> What about an environment variable BUILD_DOCX? Then if [ $BUILD_DOCX = "true" ], we build the DOCX. Therefore users can activate the feature by running

I like this idea. Gives the flexibility if they want to not build the DOCX every time.